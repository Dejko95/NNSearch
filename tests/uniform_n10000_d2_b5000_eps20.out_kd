Max visited: 70
Avg visited: 34.2095
