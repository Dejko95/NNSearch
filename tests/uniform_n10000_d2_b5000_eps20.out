
Max visited: 11
Avg visited: 3.4612306153076537
Max error: 0.1623784218169304
Avg err: 8.263887112330172E-4
Precision: 98 %
